pipeline/06-upload.R

```diff
@@ -0,0 +1,320 @@
+#- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+# 1. Setup ---------------------------------------------------------------------
+#- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+# NOTE: This script requires CCAO employee access. See wiki for S3 credentials
+# setup and multi-factor authentication help
+
+# Load libraries and scripts
+suppressPackageStartupMessages({
+  library(arrow)
+  library(aws.s3)
+  library(aws.ec2metadata)
+  library(dplyr)
+  library(glue)
+  library(here)
+  library(knitr)
+  library(lubridate)
+  library(paws.analytics)
+  library(paws.application.integration)
+  library(tidyr)
+  library(yaml)
+})
```

Comment:
Dependencies have been cleaned up here so that we only import what we need for this pipeline stage.